lib/js/components/PopOver/PopOver.vue

```diff
@@ -0,0 +1,222 @@
+			border-width: 0 $space-xs $space-s $space-xs;
+			left: calc(50% - #{$space-xs});
+			margin-bottom: 0;
+			margin-top: 0;
+			top: -$space-s;
+		}
+	}
+
+	&[x-placement^='top'] {
+		margin-bottom: $space-s;
+
+		::v-deep .popper__arrow {
+			border-color: $color-inverted-border transparent transparent transparent;
+			border-width: $space-s $space-xs 0 $space-xs;
+			bottom: -$space-s;
+			left: calc(50% - #{$space-xs});
+			margin-bottom: 0;
+			margin-top: 0;
+		}
+	}
+
+	&[x-placement^='right'] {
+		margin-left: $space-s;
+
+		::v-deep .popper__arrow {
+			border-color: transparent $color-inverted-border transparent transparent;
+			border-width: $space-xs $space-s $space-xs 0;
+			left: -$space-s;
+			margin-left: 0;
+			margin-right: 0;
+			top: calc(50% - #{$space-xs});
+		}
+	}
+
+	&[x-placement^='left'] {
+		margin-right: $space-s;
+
+		::v-deep .popper__arrow {
+			border-color: transparent transparent transparent $color-inverted-border;
+			border-width: $space-xs 0 $space-xs $space-s;
+			margin-left: 0;
+			margin-right: 0;
+			right: -$space-s;
+			top: calc(50% - #{$space-xs});
+		}
+	}
+
+	&__image {
+		border-top-left-radius: $radius-m;
+		border-top-right-radius: $radius-m;
+		max-width: 100%;
+	}
+
+	&__content {
+		padding: $space-s;
+		// Override popperjs styles
+		text-align: left;
+	}
+
+	&__title {
+		@include headlineS;
+
+		font-weight: bold;
+		margin-bottom: $space-xxs;
+	}
+
+	&__button {
+		margin: 0 $space-s $space-xs;
+		align-self: flex-end;
+	}
+}
+</style>
+
+<script>
+import VuePopper from 'vue-popperjs';
+import 'vue-popperjs/dist/vue-popper.css';
+import { POP_OVER_COLORS, POP_OVER_PLACEMENTS, POP_OVER_TRIGGER_ACTIONS } from './PopOver.consts';
+import DsButton, { BUTTON_SIZES, BUTTON_TYPES } from '../Buttons/Button';
+
+export default {
+	name: 'PopOver',
+	components: {
+		VuePopper,
+		DsButton,
+	},
+	props: {
+		boundariesSelector: {
+			type: String,
+			default: null,
+		},
+		triggerAction: {
+			type: String,
+			default: POP_OVER_TRIGGER_ACTIONS.CLICK,
+			validate(triggerAction) {
+				return Object.values(POP_OVER_TRIGGER_ACTIONS).includes(triggerAction);
+			},
+		},
+		placement: {
+			type: String,
+			default: POP_OVER_PLACEMENTS.BOTTOM,
+			validate(placement) {
+				return Object.values(POP_OVER_TRIGGER_ACTIONS).includes(placement);
+			},
+		},
+		forceShow: {
+			type: Boolean,
+			default: false,
+		},
+		color: {
+			type: String,
+			default: POP_OVER_COLORS.DEFAULT,
+			validate(color) {
+				return Object.values(POP_OVER_COLORS).includes(color);
+			},
+		},
+		titleText: {
+			type: String,
+			default: null,
+		},
+		buttonText: {
+			type: String,
+			default: null,
+		},
+		headerImageUrl: {
+			type: String,
+			default: null,
+		},
+	},
+	data() {
+		return {
+			POP_OVER_COLORS: Object.freeze(POP_OVER_COLORS),
+			BUTTON_TYPES: Object.freeze(BUTTON_TYPES),
+			BUTTON_SIZES: Object.freeze(BUTTON_SIZES),
+			key: 1,
+		};
+	},
+	watch: {
+		triggerAction() {
```

Comment:
Maybe we can make it a mixin?

Reply:
we have different prop names to be watched in these components. Having a mixin just for a single property watch is an overkill. 
Do you have alternative solution in your mind?

Reply:
No. Just popped into my mind :)